docs/installer/lokoctl.md

```diff
@@ -7,23 +7,29 @@ which includes installers for various platforms and Lokomotive components.
 
 ### Download official releases
 
-To install, find the [appropriate
-package](https://github.com/kinvolk/lokomotive/releases) for your system and
-download it. lokoctl is packaged as a `tar.gz` archive.
+Every release of Lokomotive provides the `lokoctl` binary for several operating systems.
+These binaries can be manually downloaded and installed.
 
-```console
-wget <URL-TO-DOWNLOAD-LOKOCTL>
+1. Download your [desired version](https://github.com/kinvolk/lokomotive/releases), including the GPG
+   signature.
+
+2. Verify the signature. It should match one of the [Trusted
+   keys](https://github.com/kinvolk/lokomotive/blob/master/docs/KEYS.md).
+
+```
+gpg --verify lokoctl_0.1.0_linux_amd64.tar.gz.sig
 ```
-After downloading, untar the package. Lokoctl runs as a single binary named `lokoctl`.
 
-```console
-tar xvzf <LOKOCTL_TAR_GZ_ARCHIVE>
+3. Unpack it
+
+```
+tar xvf lokoctl_0.1.0_linux_amd64.tar.gz
 ```
 
-The final step is to make sure that the `lokoctl` binary is available on the `PATH`.
+4. Find the lokoctl binary in the unpacked directory and move it to its desired location
 
-```console
-export PATH=$PATH:/path/to/lokoctl/binary
+```
+mv lokoctl_0.1.0_linux_amd64/lokoctl /usr/local/bin/lokoctl
```

Comment:
Non-privileged users don't usually have access to write to `/usr/local/bin`. Should we maybe suggest `~/.local/bin`?